backend/app/controllers/internal/companies/dividend_computations_controller.rb

```
@@ -0,0 +1,70 @@
+
+  def show
+    authorize @dividend_computation
+    render json: {
+      success: true,
+      data: DividendComputationPresenter.new(@dividend_computation).props
+    }
+  end
+
+  def create
+    authorize DividendComputation.new(company: Current.company)
+    
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)